python_modules/dagster-graphql/dagster_graphql/schema/resources.py

```diff
@@ -68,12 +65,62 @@ class Meta:
 
 
 class GrapheneJobAndSpecificOps(graphene.ObjectType):
+    # deprecated expensive fields
     job = graphene.NonNull(GrapheneJob)
     opsUsing = graphene.Field(non_null_list(GrapheneSolidHandle))
 
+    jobName = graphene.NonNull(graphene.String)
+    opHandleIDs = graphene.Field(non_null_list(graphene.String))
+
     class Meta:
         name = "JobWithOps"
 
+    def __init__(
+        self,
+        entry: ResourceJobUsageEntry,
+        location_name: str,
+        repository_name: str,
+    ):
+        self._entry = entry
+        self._location_name = location_name
+        self._repository_name = repository_name
+
+        self._cached = None
+
+    def resolve_jobName(self, _) -> str:
+        return self._entry.job_name
+
+    def resolve_opHandleIDs(self, _) -> List[str]:
+        return [handle.to_string() for handle in self._entry.node_handles]
+
+    def resolve_job(self, graphene_info: ResolveInfo):
+        if self._cached is None:
+            self._cached = self._construct_job_and_specific_ops(graphene_info)
+
+        return self._cached[0]
+
+    def resolve_opsUsing(self, graphene_info: ResolveInfo):
+        if self._cached is None:
+            self._cached = self._construct_job_and_specific_ops(graphene_info)
+
+        return self._cached[1]
```

Comment:
not strictly needed anymore now that we are no longer using the fields right? just in case others are in the future? still under test?

Reply:
just cross push safety, everything deleted in stacked pr